service/auth_status.go

```diff
@@ -205,6 +209,11 @@ func (s *AuthStatus) AuthStreamHandler(stream network.Stream) {
 			PeerID:   peerID,
 		})
 	}
+	if !confirmed && !isBlocked && autoAccept {
+		defer func() {
+			s.AddPeer(context.Background(), remotePeer, authPeer.Name, "", true)
+		}()
+	}
 
 	authResponse := protocol.AuthPeerResponse{Confirmed: confirmed, Declined: isBlocked}
```

Comment:
But the answer will say confirmed false, is that clear enough

Reply:
It may seem strange, but this situation is the same as if peer2 received a request, said "no" to peer1, and then later (after function exits) peer2 reconsidered and called to peer1 and said "yes". This was a bit easier to implement than adding peer to config immediately (synchronously) in `AuthStreamHandler` and is fully valid by protocol.

Reply:
kk if you say so